### PR TITLE
Add new book button overlapping fixed

### PR DESCRIPTION
**Pull request by akshatsemwal146**
Targets #2915 
### Description
In mobile, on the book module theadd new button is overlapping so, i fixed it.
### Checklist:
- [X] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by GauravGusain98** [comment]
I think this is not looking correct.
1. The font size of `Books(count)` can be reduced so that the button text should come in one line
2. The margin below the `Books(count)` can be reduced 
4. Also the button text font size can be reduced so that the button size does not become too large
5. The margin bottom between search and sort fields can be reduced in mobile screens. Currently they are too far away
6. The space on the right side of books card is greater then left side. So the card can be placed in between for better UI
image
image

**Comment by Hemrajbharti123** [comment]
### Local Testing Feedback
The button is still overlapping 

image

**Comment by Hemrajbharti123** [comment]
Decrease the size of `short book by category ` 


image

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 1]
```diff
@@ -13,17 +13,17 @@
...
-        <div class="col-6">
-            <h1>Books({{count($books)}})</h1>
+        <div class="col-4">
+            <h3>Books({{count($books)}})</h3    >
         </div>
         @can('library_books.create')
-        <div class="col-6">
-            <a href="{{ route('books.create') }}" class="btn btn-success float-right"><i class="fa fa-plus mr-1"></i> Add New Book</a>
+        <div class="col-8 xs-6">
+            <a href="{{ route('books.create') }}" class="btn btn-success float-right xs-2"><i class="fa fa-plus mr-1"></i> Add New Book</a>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:21]: What are the means of this class `xs-2`? I think this class is not working.
The correct class is this `col-xs-2`
```suggestion
            <a href="{{ route('books.create') }}" class="btn btn-success float-right xs-2"><i class="fa fa-plus mr-1"></i> Add New Book</a>
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
## Testing Feedbacks

**Current Behaviour**
- As far as I can see, the `Books` and the `Search` bar are not aligned, please take refer below screenshot.
image
&#8204;

**Expected Behaviour**

image

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 2]
```diff
@@ -13,17 +13,17 @@
     <br>
     <br>
     <div class="row">
-        <div class="col-6">
-            <h1>Books({{count($books)}})</h1>
+        <div class="col-4 col-xs-6 pl-4">
+            <h4>Books({{count($books)}})</h4>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/index.blade.php:17]:
```suggestion
            <h3>Books({{count($books)}})</h3>
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
One small comment. else looks good to me.

**Review by Ayush8923 — APPROVED** [review]

**Pull request merged**